lib/data/multipleBackendGateway.js

```diff
@@ -55,14 +55,17 @@ const multipleBackendGateway = {
                     key,
                     dataStoreName: controllingLocationConstraint,
                     dataStoreType: client.clientType,
+                    dataStoreVersionId,
                 };
-                if (eTag) {
-                    dataRetrievalInfo.eTag = eTag;
-                }
                 return callback(null, dataRetrievalInfo);
             });
     },
 
+    head: (objectGetInfo, reqUids, callback) => {
+        const client = clients[objectGetInfo.dataStoreName];
```

Comment:
check if client has head call and if not, just cb

Reply:
updated